README.md

```diff
@@ -1,2 +1,119 @@
-# commit.dev-proto
```

Comment:
We should keep this title

Reply:
Hi Julie, thanks for the feedback, and I have removed the "Introduction" as the title and put it back in as  #commit.dev-proto

Reply:
Repo has recently changed names. This should be updated to reflect (should be `commit.dev` now)